packages/website/src/pages/docs/Changelog/v0.8.6.mdx

```diff
@@ -0,0 +1,42 @@
+## 0.8.6
```

Comment:
No mention of imports/exports in this changelog?

Reply:
Also, I don't think this includes Version Pinning, which was a major thing. Did that go in any of these packages, or was that just separate? Could be good to include in the release notes somehow.

Reply:
Ahh, sorry, already merged. This is auto-generated, but we can improve it later. For now I'm mostly just trying to fix our new standard release workflow.

Reply:
This PR also wasn't added. Maybe summarize the comment, putting it into GPT or something? https://github.com/quantified-uncertainty/squiggle/pull/2290

Reply:
Uh... I guess I should've checked if there were any significant PRs that were missing changesets.

Reply:
Done